<launch>
    <include file="$(find scout_bringup)/launch/scout_mini_robot_base.launch"/>

	<node pkg="mavros" type="mavros_node" name="mavros" output="screen">
		<param name="fcu_url" value="/dev/ttyACM0:57600" />
	</node>

    <param name="autopilot/speed_factor_lin_x" value="1.0"/>
    <param name="autopilot/speed_factor_ang_y" value="1.0"/>

    <param name="autopilot/k_lat" value="111194"/>
    <param name="autopilot/k_long" value="50519"/>

    <param name="autopilot/vel_topic" value="cmd_vel"/>
    <param name="autopilot/rc_topic" value="scout_rc_status"/>

    <node pkg="bunker_autopilot" type="rcout_to_cmd_vel.py" name="rcout_to_cmd_vel"/>
    <node pkg="bunker_autopilot" type="rc_state_messenger.py" name="rc_state_messenger"/>
    <node pkg="bunker_autopilot" type="rc_arm_disarm.py" name="rc_arm_disarm"/>

    <node pkg="bunker_autopilot" type="mission_server.py" name="mission_server" output="screen"/>
    <node pkg="bunker_autopilot" type="mission_client.py" name="mission_client" output="screen"/>


</launch>


